adapter/denyChecker/adapter.go

```diff
@@ -15,59 +15,29 @@
 package denyChecker
 
 import (
-	"fmt"
-
 	"github.com/golang/protobuf/proto"
 	"google.golang.org/genproto/googleapis/rpc/code"
 
 	"istio.io/mixer/pkg/aspect/denyChecker"
 	"istio.io/mixer/pkg/aspectsupport"
 )
 
-const (
-	// ImplName is the canonical name of this implementation
-	ImplName = "istio/denyChecker"
-)
-
-// Register records the existence of this adapter
 func Register(r aspectsupport.Registry) error {
 	return r.RegisterDeny(newAdapter())
 }
 
 type adapterState struct{}
 
-func newAdapter() denyChecker.Adapter {
-	return &adapterState{}
-}
-
-func (a *adapterState) Name() string {
-	return ImplName
-}
-
-func (a *adapterState) Description() string {
-	return "Deny every check request"
-}
+func newAdapter() denyChecker.Adapter                          { return &adapterState{} }
+func (a *adapterState) Name() string                           { return "istio/denyChecker" }
```

Comment:
i don't know that there is an overwhelming need. This may be a personal prefs thing (i like having the name/desc consts at the top of the adapters as sort of an easily discoverable, readable, self-documenting thing, but I can see the argument against).

Does the name need to be exported or referenced directly?  Before, it was exported.